autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -117,12 +117,6 @@ variable "ip_range_services" {
   description = "The _name_ of the secondary subnet range to use for services"
 }
 
-variable "initial_node_count" {
```

Comment:
Having this exposed allows us to right size the control plane. b/167711149 for context

Reply:
Got it - as an alternative approach we can set to `1` if `0` is supplied.